### ISSUE TITLE
TeX Extension Options

### ISSUE THREAD

**Issue by colinc86** [colinc86/MathJaxSwift#21]
Implement [TeX extension options](https://docs.mathjax.org/en/latest/options/input/tex.html#tex-extension-options-1).

**Comment by juancazalla** [comment]
Hi @colinc86! 

Great work in the library. Could you guide me on how I can help with this? I would like to add support for some AMSmath labels like `\binom`

**Comment by colinc86** [comment]
Hey thanks, @juancazalla, and sorry for not getting around to this sooner!

You can load different extensions through the `TexInputProcessorOptions`' `loadPackages` property. Only the `base` extension is loaded by default, so this will error:

```swift
let output = try await mathjax.tex2svg("\\binom{n}{k}")
```

But this will render:

```swift
let texOptions = TeXInputProcessorOptions(loadPackages: [TeXInputProcessorOptions.Packages.ams])
let output = try await mathjax.tex2svg("\\binom{n}{k}", inputOptions: texOptions)
```

If you want all of the extensions available to MathJax loaded you can use `TeXInputProcessorOptions.Packages.all` as the input array.
